examples/control-elements/media-chrome-menu-button.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta name="viewport" content="width=device-width">
+  <title>Media Chrome Captions Usage Examples</title>
+  <script type="module" src="../../dist/index.js"></script>
+  <script type="module" src="../../../dist/media-chrome-listbox.js"></script>
+  <script type="module" src="../../../dist/media-chrome-listitem.js"></script>
+  <script type="module" src="../../../dist/media-captions-listbox.js"></script>
+  <script type="module" src="../../../dist/media-chrome-menu-button.js"></script>
+  <style>
+    media-captions-listbox {
```

Comment:
Really not loving the growing examples that require CSS to work. I can see myself using this component directly, so I don't think the same reasoning applies here as in the other conversation.

Reply:
Default styles is a follow-up that I'll be tackling next. See https://github.com/muxinc/media-chrome/issues/387 before we call this whole thing complete.

Reply:
This component in particular definitely showed me that we do need default styles.